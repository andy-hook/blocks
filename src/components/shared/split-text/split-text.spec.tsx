import React from "react"

import SplitText from "./split-text"
import { renderWithTheme } from "@test-utils"

jest.mock("gsap")

describe("<SplitText />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <SplitText visible={true} animate={false}>
        This is test text
      </SplitText>
    )

    expect(tree).toMatchSnapshot()
  })

  it("renders without widow if specified", () => {
    const tree = renderWithTheme(
      "dark",
      <SplitText visible={true} animate={false} removeWidow={false}>
        This is test text
      </SplitText>
    )

    expect(tree).toMatchSnapshot()
  })
})
