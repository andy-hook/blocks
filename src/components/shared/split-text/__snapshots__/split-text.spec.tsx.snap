// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplitText /> renders correctly 1`] = `
Array [
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      T
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      h
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      i
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      s
    </div>
     
  </div>,
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      i
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      s
    </div>
     
  </div>,
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      <div
        className="sc-bdVaJa kaUtgC"
      >
        t
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        e
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        s
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        t
      </div>
       
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      <div
        className="sc-bdVaJa kaUtgC"
      >
        t
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        e
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        x
      </div>
      <div
        className="sc-bdVaJa kaUtgC"
      >
        t
      </div>
      
    </div>
  </div>,
]
`;

exports[`<SplitText /> renders without widow if specified 1`] = `
Array [
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      T
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      h
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      i
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      s
    </div>
     
  </div>,
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      i
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      s
    </div>
     
  </div>,
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      t
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      e
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      s
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      t
    </div>
     
  </div>,
  <div
    className="sc-bdVaJa kaUtgC"
  >
    <div
      className="sc-bdVaJa kaUtgC"
    >
      t
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      e
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      x
    </div>
    <div
      className="sc-bdVaJa kaUtgC"
    >
      t
    </div>
    
  </div>,
]
`;
